### DISCUSSION TITLE
Accidentally used Move Domain on Third-Party domain instead of Transfer Domain

### DISCUSSION THREAD

**Discussion by CrimsonWitness** [vercel/community#4828] [Help] [answered]
### Summary

Hey there, new to using Vercel to manage domains.

My account has only one project at the moment and I tried to add a domain to it which I have registered through Namecheap.

This added my domain, `crimsonwitnesses.com` to the domains list for my Vercel team.

After that, I meant to Transfer that domain from Namecheap to Vercel, but I accidentally used the Move command under the domains list instead. This only provided me one option, the personal account I'm logged in with. But after that was completed, the domain is no longer listed under my team's domains list and there doesn't seem to be a way I can see domains for the personal account at all.

Even using the Vercel CLI, I'm seeing zero domains for *both* the personal account *and* the team. Yet when I try to use the Transfer domain flow, it tells me that the domain already exists.

Here's what I got from the CLI output:
![image](https://github.com/vercel/community/assets/150510174/723f7c3e-4bf1-47fe-82f0-b944802221e5)

My activity log shows that the domain was moved... somewhere:
![image](https://github.com/vercel/community/assets/150510174/7aa7bd70-192a-4531-ae40-118fb3eff067)

The domain itself is still registered with Namecheap, and I was able to add the correct TXT and A records to get the domain working with my project, but I'm still stuck with this problem of not being able to transfer the domain to Vercel. I'm guessing this has to be an error somewhere on Vercel's end, because from my dashboard my domains list is empty and there doesn't seem to be any way I can reverse this move action now.

![image](https://github.com/vercel/community/assets/150510174/b138ef1b-9cdf-4489-8d35-e93241ae2db4)

Any help resolving this would be appreciated!

### Steps to Reproduce

Created new account
Created new team
Added new project
Added third-party domain to project
Moved domain from the domains dashboard (presumably to the account? as it was the only option)
Domain is now missing

**Comment by amyegan** [comment, marked as answer]
Hi @CrimsonWitness. Sorry you ran into this problem. Looks like the domain did end up in the wrong place after that transfer, so I removed it from there for you. Can you try re-adding the domain and let me know if that works?

**Reply by CrimsonWitness** [reply]
This appears to have solved the issue! I no longer get an error when trying to transfer in the third-party domain. Thanks for investigating this @amyegan!

**Reply by joelangeway** [reply]
@amyegan I am having the same problem with joelangeway.com . Would you please do this same fix for me?

**Reply by amyegan** [reply]
Hey @joelangeway. It looks like the domain is currently associated with your Hobby team. You should be able to see the domain on the `Domains` tab of your team dashboard. 

https://vercel.com/dashboard/domains

**Reply by joelangeway** [reply]
Amy, Thank You! I finally have my domain transfer in progress.
